puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/foreman-packaging-pr-test.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- job:
+    name: foreman-packaging-pr-test
```

Comment:
pr-test-rpm?

Reply:
I'd prefer the `rpm` part being earlier, something like `foreman-packaging-rpm-pr-test` to stay in line with conventions we established on the mailing list.

Reply:
k